src/pmdas/opentelemetry/pmdaopentelemetry.python

```diff
@@ -445,19 +507,30 @@ class Source(object):
             help_text = ''
         return help_text, help_oneline
 
-    def parse_metric_line(self, metric_dict):
-        mname = metric_dict['name']
+    def collect_attributes(self, attribute_list):
+        labels = {}
+        for item in attribute_list:
+            key = item["key"]
+            value = item["value"]["stringValue"]
+            labels[key] = value
+        return labels
+    
+    def pcp_name(self, name):
+        return name.replace(".", "_")
+    
```

Comment:
@lmchilton let's drop this pcp_name routine - reading more about OpenTelemetry, I think we can use PCP names unchanged and dotted notation is in fact a more fitting style there.  I updated pmproxy earlier in this way also.